### ISSUE TITLE
Ubuntu 18.04 no bin folder after build. 

### ISSUE THREAD

**Issue by MrEMC** [hirschmann/nbfc#506]
I have issues trying to build version 1.5.0. I don't have bin file in linux and it also seem to include new files.

I am getting these warrning which I assume should be alright. 
![image](https://user-images.githubusercontent.com/28583659/41914238-c6746278-7995-11e8-8b44-8c3e22724048.png)

The directory after build. 
![image](https://user-images.githubusercontent.com/28583659/41914198-a97bf410-7995-11e8-8e1c-3ed2a61d905e.png)

**Comment by TobiasKuch** [comment]
i have the exact same Problem on Ubuntu 18.04

**Comment by egourgoulhon** [comment]
I also had the same issue with Ubuntu 18.04 running on a HP Zbook G3. It turns out that the build failed because the `mono-complete` package shipped with Ubuntu 18.04 is too old (this is version 4.6.2) to build nbfc. The solution is to install the latest `mono-complete` package from the [Mono project page](https://www.mono-project.com/download/stable/) (version 5.14.0), as follows:
```
sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
echo "deb https://download.mono-project.com/repo/ubuntu stable-bionic main" | sudo tee /etc/apt/sources.list.d/mono-official-stable.list
sudo apt update
sudo apt install mono-complete
```
Then you can do
```
git clone https://github.com/hirschmann/nbfc.git
cd nbfc
./build.sh
sudo cp -r Linux/bin/Release/ /opt/nbfc
sudo cp Linux/nbfc.service /etc/systemd/system/
sudo cp Linux/nbfc-sleep.service /etc/systemd/system/
systemctl enable nbfc --now
cd /opt/nbfc
sudo chmod +x nbfc.exe
./nbfc.exe config --apply "HP ZBook 15"  # replace by your version
```
and everything works. 
Side notes:
- the name of the directory is  `Linux/bin/Release` and not `Linux/bin/ReleaseLinux` as written in the _installation_ section of the [First steps page](https://github.com/hirschmann/nbfc/wiki/First-steps).
- the built file `nbfc.exe` was not in executable mode and I had to do `sudo chmod +x nbfc.exe` before running `nbfc.exe config --apply`.
